modules/cnv-confirming-policy-updates-on-nodes.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-confirming-policy-updates-on-nodes_{context}"]
+= Confirming Policy updates on nodes
+
+A `NodeNetworkConfigurationPolicy` manifest describes your desired network configuration for nodes in the cluster.
+When applied, the cluster creates a `NodeNetworkConfigurationPolicy` object for the whole cluster.
+The Policy object includes your desired network configuration and the actual status of the Policy on the cluster as a whole.
+
+
+When a Policy is created, a `NodeNetworkConfigurationEnactment` object is created on every node in the cluster. The Enactment is a read-only object that represents the desired state of the Policy and the actual status of the Policy on that node. 
```

Comment:
It is not created on node (it is just an object in database), maybe "per each node"?

We can drop "represents the desired state of the Policy". We have not implemented any templating, so desiredState in Enactment is exactly the same as the one for Policy. So it is not interesting for users.

Reply:
Updated to:
"When you apply a Policy, a `NodeNetworkConfigurationEnactment` is created for every node in the cluster. The Enactment is a read-only object that represents the status of execution of the Policy on that node.       
If the Policy fails to be applied on the node, the Enactment for that node includes a traceback that can help with troubleshooting."

Reply:
Updated to:
"When you apply a Policy, a `NodeNetworkConfigurationEnactment` is created for every node in the cluster. The Enactment is a read-only object that represents the status of execution of the Policy on that node.
If the Policy fails to be applied on the node, the Enactment for that node includes a traceback that can help troubleshoot."